### ISSUE TITLE
Tunable consistency as a config parameter

### ISSUE THREAD

**Issue by hopugop** [scylladb/scylla-migrator#80]
Currently consistency can be tuned by passing config Spark config parameters to the submit job. This could be a configurable option in source/target entries on the config file.

Examples of Spark submit parameters:
```shell
--conf spark.cassandra.input.consistency.level=LOCAL_QUORUM 
--conf spark.cassandra.output.consistency.level=LOCAL_QUORUM 
```

The default consistency for input is LOCAL_ONE, while output is LOCAL_QUORUM. On a cluster that does not run repairs, this means data can be inconsistent among different runs.

**Comment by netguy204** [comment]
yes! And the example config should be updated to mention these new parameters as well.

**Comment by hopugop** [comment]
cc @tarzanek
